spec/support/shared_examples/behaves_like_financial_value.rb

```diff
@@ -0,0 +1,61 @@
+RSpec.shared_examples "a financial value" do |header|
```

Comment:
Could we have an explicit spec for "when value is 0", so we're clear what happens?